auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/RestWrapper.java

```diff
@@ -14,6 +14,7 @@
 
 package com.wipro.ats.bdre.md.auth;
 
+import org.apache.log4j.Logger;
 import org.codehaus.jackson.annotate.JsonProperty;
 
 import java.lang.reflect.InvocationTargetException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.lang.reflect.InvocationTargetException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)